src/igl/vulkan/Device.h

```diff
@@ -37,40 +37,40 @@ class Device final : public IDevice {
 
   // Command Queue
   std::shared_ptr<ICommandQueue> createCommandQueue(const CommandQueueDesc& desc,
-                                                    Result* outResult) override;
+                                                    Result* IGL_NULLABLE outResult) override;
```

Comment:
this PR got rejected intenaly since it's only updating the .h and not the .cpp.

do you think you can make that update and resubmit?